### PR TITLE
"/latest" in hrefs is redundant

### PR DESCRIPTION
**Pull request by hmmr**
resulting in affected links (four, in this document) being broken.

### CODE REVIEW

File: source/languages/en/riak/dev/advanced/replication-properties.md [hunk 1]
```diff
@@ -92,8 +92,8 @@ Parameter | Common name | Default value | Description
 `n_val` | N | `3` | Replication factor, i.e. the number of nodes in the cluster on which an object is to be stored
 `r` | R | `quorum` | The number of servers that must respond to a read request
 `w` | W | `quorum` | Number of servers that must respond to a write request
-`pr` | PR | `0` | The number of primary <a href="/latest/theory/concepts/vnodes">vnodes</a> that must respond to a read request
-`pw` | PW | `0` | The number of primary <a href="/latest/theory/concepts/glossary#vnode">vnodes</a> that must respond to a write request
+`pr` | PR | `0` | The number of primary <a href="/theory/concepts/vnodes">vnodes</a> that must respond to a read request
+`pw` | PW | `0` | The number of primary <a href="/theory/concepts/glossary#vnode">vnodes</a> that must respond to a write request
```
Comment (lucperkins) [inline source/languages/en/riak/dev/advanced/replication-properties.md]: Slight nitpick: this should use the same URL as the previous example, i.e. to [this doc](http://docs.basho.com/riak/latest/theory/concepts/vnodes/). The glossary entry doesn't provide much info.

File: source/languages/en/riak/dev/advanced/replication-properties.md [hunk 2]
```diff
@@ -524,8 +524,8 @@ vnodes responsible for an object.
 Scenario | What happens in Riak
 :--------|:--------------------
 All 3 vnodes agree on the value | Once the first 2 vnodes return the value, that value is returned to the client
-2 of 3 vnodes agree on the value, and those 2 are the first to reach the coordinating node | The value is returned returned to the client. Read repair will deal with the conflict per the later scenarios, which means that a future read may return a different value or <a href="/latest/theory/concepts/vector-clocks#siblings">siblings</a>
-2 conflicting values reach the coordinating node and <a href="/latest/theory/concepts/vector-clocks">vector clocks</a> allow for resolution | The vector clocks are used to resolve the conflict and return a single value, which is propagated via read repair to the relevant vnodes
+2 of 3 vnodes agree on the value, and those 2 are the first to reach the coordinating node | The value is returned returned to the client. Read repair will deal with the conflict per the later scenarios, which means that a future read may return a different value or <a href="/theory/concepts/vector-clocks#siblings">siblings</a>
```
Comment (lucperkins) [inline source/languages/en/riak/dev/advanced/replication-properties.md]: Actually, these links should be both be changed to `/theory/concepts/context#Siblings`

Comment (TheOnlyRew) [inline source/languages/en/riak/dev/advanced/replication-properties.md]: Shouldn't these be markdown-style links, rather than HTML anchors?

Comment (lucperkins) [inline source/languages/en/riak/dev/advanced/replication-properties.md]: Unfortunately, Markdown-style links don't work in tables

**Comment by kesslerm** [comment]
+1

**Pull request merged**